client/src/components/profile/components/portfolio-projects.tsx

```diff
@@ -19,7 +18,7 @@ export const PortfolioProjects = ({
     return null;
   }
   return (
-    <FullWidthRow>
+    <>
```

Comment:
This change is out of scope for this pr. When reverting this change, please alter the css accordingly to make sure the button does not go out of the card.

Reply:
This change is my suggestion, if it is reverted it would break the layout in medium layout

Reply:
In that case, LGMT. The layout of the card can improved further as there are a lot of unused white space. If you are interested we could tackle it in a separate issue/pr.